### ISSUE TITLE
create transcript creation guide for submitting tickets

### ISSUE THREAD

**Issue by aryairani** [unisonweb/unison#4913]

**Comment by sellout** [comment]
Just to expand this a bit, for a newbie like myself – is this about having a document that is linked to from the GitHub issue template to encourage reporters to write up issues in template style? And I guess the same thing is even more useful for reporters on Share.

Tangential: if someone opens an issue where the content is a valid transcript, how does one then extract & run that transcript? The closest thing I can figure is you “Quote reply” the description, then copy the quoted bit and strip the leading `> ` from it. Is there a better way? Or a way for `ucm` to accept “GitHub transcripts” (i.e., `ucm` stripping the `> ` internally).

**Comment by aryairani** [comment]
Yeah, a template or a guide, so we can ask people to submit transcripts when they write up issues, as a shortcut to reproducing the issue.

It's a little tricky because we have "transcripts" and also "transcript output". Depending on the issue we might just need a transcript, or we might want them to include their output too. Currently transcript output isn't a superset of the original transcript, so we'd need some way to figure out which is which.

What I've sometimes done recently is include the transcript and transcript output quoted in the ticket, and then also link the transcript as a gist.  Then you can reproduce it by downloading a zip of the gist from github, unzipping it, and passing it to ucm.

We could imagine some ways to try to automatically load transcripts from Github, but no we don't have anything now.
